internal/testhelpers/e2e/session.go

```diff
@@ -562,8 +562,16 @@ func (s *Session) Close() error {
 		}
 	}
 
+	serviceURL := api.GetServiceURL(api.ServiceMono)
+	host := os.Getenv(constants.APIHostEnvVarName)
+	if host == "" {
+		host = constants.DefaultAPIHost
+	}
+	serviceURL.Host = strings.Replace(serviceURL.Host, string(api.ServiceMono)+api.TestingPlatform, host, 1)
+
 	for _, proj := range s.createdProjects {
-		err := model.DeleteProject(proj.Owner, proj.Project, auth)
+		params := projects.NewDeleteProjectParams().WithOrganizationName(proj.Owner).WithProjectName(proj.Project)
+		_, err := mono.Init(serviceURL, nil).Projects.DeleteProject(params, auth.ClientAuth())
```

Comment:
Would you please explain why this change is necessary? I can't grok it :(

Reply:
I believe that I was running into an issue where the test suite was trying to delete a project form a `testing.tld` domain... let me try to remove it and see what happens.